samples/SampleApp/SampleApp.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web" ToolsVersion="15.0">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net451;netcoreapp1.1</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <!-- TODO remove rid when https://github.com/dotnet/sdk/issues/396 is resolved -->
+    <RuntimeIdentifier Condition="'$(TargetFramework)'!='netcoreapp1.1'">win7-x64</RuntimeIdentifier>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="testCert.pfx">
```

Comment:
I commented on the commit but I'll comment here too 😄. Change these to attributes. Also `CopyToOutputDirectory` infers `CopyToPublishDirectory`.

```XML
<Content Include="testCert.pfx" CopyToOutputDirectory="PreserveNewest" />
```